content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -233,7 +233,7 @@ You can install kubectl as part of the Google Cloud SDK.
     sudo mv ./kubectl /usr/local/bin/kubectl
     ```
 {{% /tab %}}
-{{% tab  name="Windows" %}}
+{{% tab name="Windows" %}}
```

Comment:
Does the extra space cause a problem?

Reply:
Please let me know if you need modification

Reply:
It is okay.